### DISCUSSION TITLE
Reduce the recoil from damaging effects when using Quantum Armor

### DISCUSSION THREAD

**Discussion by Enhanced007** [TechReborn/TechReborn#3138] [Ideas]
When using the armor and in flight there is way too much knock back when damage is received. When flying, any damage taken whether it is a wither effect, hunger, or the annoying phantoms, it causes you to immediately fall down around 4 blocks and this is a pain to fight against. When fighting the Wither it is almost impossible to get to him unless it's in an enclosed space because every second you're sinking lower and lower.

**Comment by Ayutac** [comment]
Any idea how to archive this?

**Comment by drcrazy** [comment]
You can close this as outdated
